ntex-server/src/signals.rs

```diff
@@ -1,9 +1,12 @@
+use std::sync::atomic::{AtomicBool, Ordering};
 use std::{cell::RefCell, thread};
 
 use ntex_rt::System;
 
 use crate::server::Server;
 
+static CTRL_C_HANDLER_SET: AtomicBool = AtomicBool::new(false);
```

Comment:
could you mark it with #[cfg(target_family = "windows")]

Reply:
I found this fix can't stop the ntex-server normally. I need some time to find a other way. And I will add  #[cfg(target_family = "windows")]

Reply:
you should store System in global before checking for var and use it instead of System::current()

Reply:
> you should store System in global before checking for var and use it instead of System::current()

I'm not very good at reading all this complex runtime management code, I'm afraid I need someone who knows more about ntex to fix this problem 😳